### ISSUE TITLE
Allow users to rearrange reciters

### ISSUE THREAD

**Issue by benomaire** [quran/quran_android#1448]
I assume this will be more needed as more reciters are added. I think allowing "rearranging" in the settings should be enough, and no "hiding" should be allowed. What do you think?

From play store:

- 17 mars 2020
>  salam , très bonne application ce serait mieux si on pouvait choisir quels récitateurs afficher dans le volet déroulant parce qu'il y en a beaucoup et ce n'est pas évident de retrouver celui qu'on cherche

**Comment by nacer80** [comment]
السلام عليكم ورحمة الله وبركاته 
I agree with you, it's a good suggestion to add option of reciters rearranging order,  same like tafseer and translation.

**Comment by ahmedre** [comment]
وعليكم السلام ورحمة الله وبركاته
for reciters, i want to do something different, which is mark which suras are downloaded for that sheikh and maybe grouping them at the top.

**Comment by benomaire** [comment]
Good idea, but what about people using streaming? A history of previously used reciters?
